survey_formio/models/survey.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2023 Acsone SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+from datetime import datetime, timedelta
+
+from odoo import models
+
+
+class SurveySurvey(models.Model):
+    _inherit = "survey.survey"
+
+    def _get_formio_display(self):
+        """
+        Returns a string representing the form.io display
+        Correct values are form, wizard or pdf. At the moment only
+        form is supported.
+        In the future use questions_layout to determine display.
+        """
+        self.ensure_one()
+        return "form"
+
+    def _get_components(self, day_first=True):
+        """
+        Pages are not supported yet.
+
+        Returns a list of dict representing survey questions as form.io components
+
+        Parameters:
+        day_first: boolean that determines if the day comes first or
+                   not in dates
+        """
+        self.ensure_one()
+        components = []
+        for question in self.question_ids:
+            components.append(question._get_formio_dict(day_first))
+        return components
+
+    def generate_formio_json(self, day_first=True):
+        """
+        Returns a form.io conform json representation of a survey.
+
+        Parameters:
+        day_first: boolean that determines if the day comes first or
+                   not in dates
```

Comment:
```suggestion
        day_first: True if the day input should be the first input item for date fields
```

This is a bit strange to propage this day_first options across all methods.
Is it really the only thing that is and will be configurable?
Maybe we need an options data structure?
Or be more opinionated (say, dayFirst=true always) and leave configurability for later when we have concrete use cases?

Reply:
Does dayFirst influence the display of the form, or the datetime format in the answer, or both?

Reply:
It influences both. Without setting it I can't know which date format I'll receive. For now I think it's the only configurable thing. 
Should I just say dayFirst=True for now then?

Reply:
> It influences both. Without setting it I can't know which date format I'll receive. For now I think it's the only configurable thing.

That is surprising. Is there not a mechanism to tell it to return an ISO format for the dates?

> Should I just say dayFirst=True for now then?

Yes.

Reply:
> That is surprising. Is there not a mechanism to tell it to return an ISO format for the dates?
If there's one I couldn't find it. It surprised me too, especially because the datetime component always returns the same format.
> Yes.
Ok. I'll do that

Reply:
Can we use the `datetime` widget with `"enableTime": false` instead of the day widget?